students/hyska_monika/lesson_14_decorators/authorization.py

```diff
@@ -0,0 +1,14 @@
+access_count = -1
+
+
+def has_access():
+    '''
+    let's pretend this function calls some authorization services
+    returns True if access is granted and False if not
+    access is granted every 3rd time
+    '''
+    global access_count
+    access_count += 1
+    if access_count >= 3:
+        access_count = 0
+    return access_count == 0
```

Comment:
You took the instructions very literally. I didn't expect that level of conformance. 

It would be much more readable as:
```
access_count = 0
...
def ...
    access_count += 1
    return access_count in (2, 3)
```